components/pedigree/resources/src/main/resources/pedigree/view/personHoverbox.js

```diff
@@ -106,7 +106,7 @@ define([
                     }
                     // parent handle
                     this.generateHandle('parent', x, splitLocationY, x, y - PedigreeEditorParameters.attributes.personHandleLength,
-                                        "Click to create new nodes for the parents or drag to an existing person or partnership (valid choices will be highlighted in green). Dragging to a person will create a new relationship.", "F", topHandleHint);
+                                        "Click to draw parents or drag to an existing person or partnership (valid choices will be highlighted in green). Dragging to a person will create a new relationship.", "F", topHandleHint);
```

Comment:
just curious, in the context of creating pedigree nodes, we want to use the word "draw" wherever we used to say "create"?

Reply:
"Create" is a word that we, devs, tend to use because we know we're instantiating an object. Our target users are accustomed to "drawing" pedigrees.

Reply:
That was my question - so I see that for them new stuff is "drawn"